charts/telegraf-sidecar-operator/values.yaml

```diff
@@ -43,6 +43,9 @@ operator:
           nodename = "$NODENAME"
           namespace = "$NAMESPACE"
           type = "app"
+  mutatingWebhook:
+    # -- Configure the namespace selection to call the webhook
+    namespaceSelector: {}
```

Comment:
If I could request a small change, could this please me moved to the top level of the values file rather than nested under the `operator` key. So instead of `.Values.operator.mutatingWebhook.namespaceSelector` it would just be `.Values.mutatingWebhook.namespaceSelector`.

Reply:
Hi @jmickey,
I moved to the root values as requested 🙌